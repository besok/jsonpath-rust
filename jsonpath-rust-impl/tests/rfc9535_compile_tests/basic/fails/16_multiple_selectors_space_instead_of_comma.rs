//! Test case: 16_multiple_selectors_space_instead_of_comma
//! Tags: whitespace

use ::jsonpath_ast::ast::Main;
use ::jsonpath_rust_impl::json_query;

fn main() {
    assert_eq!(
        json_query!( $[0 2] ),
        Main::try_from_pest_parse("$[0 2]").expect("failed to parse")
    );
}
