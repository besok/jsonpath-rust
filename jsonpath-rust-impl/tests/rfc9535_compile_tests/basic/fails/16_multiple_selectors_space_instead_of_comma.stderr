error: expected `,`
 --> tests/rfc9535_compile_tests/basic/fails/16_multiple_selectors_space_instead_of_comma.rs:9:27
  |
9 |         json_query! ( $[0 2] ),
  |                           ^
